src/commands/synthetics/interfaces.ts

```diff
@@ -43,7 +49,7 @@ export interface BrowserTestResult extends TestResult {
     width: number
   }
   duration: number
-  error?: string | 'Endpoint Failure' | 'Timeout' | 'Tunnel Failure'
+  error?: string | ERRORS
```

Comment:
No change required: we moved away from enum to union, however in that case, having `string | 'value' | ...` defeats the purpose of the union, we should change that (and this is another point in favor of sharing synthetics typings)